packages/server-wallet/src/chain-service/types.ts

```diff
@@ -58,7 +58,30 @@ interface ChainEventEmitterInterface {
   destructor(): void;
 }
 
+export type FundChannelRequest = {type: 'FundChannel'} & FundChannelArg;
+export type ConcludeAndWithdrawRequest = {
+  type: 'ConcludeAndWithdraw';
+  finalizationProof: SignedState[];
+};
+export type PushOutcomeAndWithdrawRequest = {
+  type: 'PushOutcomeAndWithdraw';
+  state: State;
+  challengerAddress: Address;
+};
+
+export type ChallengeRequest = {
+  type: 'Challenge';
+  challengeStates: SignedState[];
+  privateKey: PrivateKey;
+};
+
+export type ChainRequest =
```

Comment:
This type seems pretty similar to the existing `ChainServiceRequest`. Probably outside the scope of this but we may want to look at refactoring them into one.